### PR TITLE
init

### PR DESCRIPTION
**Pull request by gdvl777**
- [x] Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- [x] Create a directory for your reusable components: /components.
- [x] Install [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)
- [x] [Setup your router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of <App>
- [x] Add 2 routes in your application, each should render a page component:
- [x] /
- [x] /categories
- [x] Create components inside of the /components directory:
- [x] Contains individual book state, as well as a button to delete the book
- [x] Contains a list to renders individual books
- [x] Contains a form to create new books
- [x] Contains a navigation to link to the 2 routes you created

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    setBooks((prevBooks) => prevBooks.filter((book) => book.id !== bookId));
+  };
+
+  return (
+    <Router>
+      <div>
+        <Navigation />
+        <Routes>
+          <Route path="/" element={<BookList books={books} onDelete={handleDeleteBook} />} />
+        </Routes>
```
Comment (OLIPLICHE) [inline src/App.js]: - [x] Nice job so far, I would advise you to follow what the requirements say: Categories should also be in routes and not a placeholder in Navigation as you did it
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @gdvl777, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- React-router-dom should be present in package.json 👍 
- `Books` components BookList, Book, NewBookForm 👍 

## Required Changes ♻️

- [ ] Optional: I would advise you to make your `commit` messages meaningful in the next milestone, because it favors tracing your code, it shows professionalism. That is why it is part of the general requirements 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  };
+
+  return (
+    <Router>
+      <div>
+        <Navigation />
+        <Routes>
+          <Route path="/" element={<BookList books={books} onDelete={handleDeleteBook} />} />
+          <Route path="/components" element={<BookForm />} />
+        </Routes>
```
Comment (Alusp) [inline src/App.js]: - [x] Good job so far implementing the changes requested by the previous reviewer, however, as he mentioned and requested by the requirement, `Add 2 routes in your application, each should render a page component:`
`/`
`/categories`

As indicated above, routes should include Home and Categories. Please resolve that. :slightly_smiling_face:

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @gdvl777, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
close

 ## Highlights
✔️ No linter Errors
✔️ Descriptive README
✔️ Descriptive PR title and description
✔️ Completed most of the Tasks



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ademibowale — APPROVED** [review]
Hi @gdvl777 , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️
- Contains a list to renders individual books✔️
-  Contains a form to create new books✔️
- Contains a navigation to link to the 2 routes you created✔️
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**